packages/dds/tree/src/shared-tree/schematizeTree.ts

```diff
@@ -134,24 +130,6 @@ export function canInitialize(checkout: ITreeCheckout): boolean {
 	return checkout.forest.isEmpty && schemaDataIsEmpty(checkout.storedSchema);
 }
 
-function normalizeNewFieldContent(
-	content: readonly ITreeCursorSynchronous[] | ITreeCursorSynchronous | undefined,
-): ITreeCursorSynchronous {
-	if (content === undefined) {
-		return cursorForMapTreeField([]);
-	}
-
-	if (isReadonlyArray(content)) {
-		return cursorForMapTreeField(content.map((c) => mapTreeFromCursor(c)));
-	}
-
-	if (content.mode === CursorLocationType.Fields) {
-		return content;
```

Comment:
We only use this case now, so no need for thuis function which is an identity in this case.